joplin/static/css/admin.css

```diff
@@ -1,9 +1,26 @@
 .full input {padding: 1em;}
 
-#tab-content .object .object-help {display: none;}
 .object fieldset, .object .field-content {width: 100%;}
 
 .translation-tab .object .object-help {width: auto; opacity: 1; position: static; font-size: 1.2em;}
 .translation-tab .object fieldset {padding-top: 0.5em;}
 
 .hidden {display: none !important;}
+
+.wrapper {padding-left: 60px;}
+.nav-wrapper, .nav-wrapper .inner {width: 60px;}
```

Comment:
There's something weird going on here when you scroll...
![screen shot 2018-07-10 at 4 44 59 pm](https://user-images.githubusercontent.com/5697474/42539440-c2ffb81a-8460-11e8-82fc-3e8e126a0f3f.png)